ocaml/xapi/cli_operations.ml

```diff
@@ -646,12 +646,13 @@ let make_param_funs getall getallrecs getbyuuid record class_name def_filters de
   let p_set (printer : printer) rpc session_id params =
     let record = get_record rpc session_id (List.assoc "uuid" params) in
     let set_params = List.filter (fun (p,_) -> not (List.mem p ("uuid"::stdparams))) params in
+    let set_map_params = Hashtbl.create 10 in
```

Comment:
Please add a comment that explains what exactly goes into this hash table. Also the name could be more descriptive.

A type annotation would help to understand this as well (`let ... : (..., ...) Hashtbl.t = ...`).